tests/layer_tests/pytorch_tests/test_floor_divide.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class Testfloor_divide(PytorchLayerTest):
```

Comment:
I think that following naming convention it should be `TestFloorDivide`

Reply:
Yeah, I am gonna fix it